archivist/src/main/scala/uk/ac/wellcome/platform/archive/archivist/flow/ArchiveDigestItemJobFlow.scala

```diff
@@ -14,17 +14,17 @@ import uk.ac.wellcome.platform.archive.common.models.error.ArchiveError
 object ArchiveDigestItemJobFlow extends Logging {
```

Comment:
nit: rename this flow to match the model?

```suggestion
object DigestItemJobFlow extends Logging {
```